client/src/store/search/actions.ts

```diff
@@ -42,14 +47,16 @@ export const getSearchResults = (query: string, page: number): (dispatch: Dispat
     async (dispatch: Dispatch): Promise<void> => {
         try {
             const FIRST_PAGE = 1;
+            const RESULTS_RER_PAGE = 5;
+            dispatch(getSearchResultsPending());
             const res = await AuthAPI.get(`/search/${query}/${page}`);
             if (page === FIRST_PAGE) {
                 dispatch(clearLoaded());
                 dispatch(getSearchResultsSuccess(res.data.users));
             } else {
                 dispatch(getMoreResultsSuccess(res.data.users));
             }
-            if (!res.data.users.length) {
+            if (!res.data.users.length || res.data.users.length % RESULTS_RER_PAGE !== 0) {
```

Comment:
RESULTS_PER_PAGE !== 0 ?

Reply:
I used modulo operator (%) here. So, searchResults.length % RESULTS_PER_PAGE can be 0.

RESULTS_PER_PAGE - is the number of results we show per page, in our case 5. So, if number of search results % 5 !== 0, it 100% means that all results have been loaded. So, we will dispatch this action, and based on _loaded_ flag we will show or not our waypoint (which loads more results) in our component.

Reply:
Fixed.